contracts/account/Account.sol

```diff
@@ -16,4 +18,10 @@ import {AccountCore} from "./AccountCore.sol";
  * NOTE: To use this contract, the {ERC7739Signer-_rawSignatureValidation} function must be
  * implemented using a specific signature verification algorithm. See {SignerECDSA}, {SignerP256} or {SignerRSA}.
  */
-abstract contract Account is AccountCore, ERC721Holder, ERC1155Holder, ERC7739Signer {}
+abstract contract Account is AccountCore, Multicall, ERC721Holder, ERC1155Holder, ERC7739Signer {
+    function execute(address target, uint256 value, bytes calldata data) public virtual onlyEntryPointOrSelf {
```

Comment:
What's the rationale of having another `execute` function if the Account already has `executeUserOp`?

Reply:
`executeUserOp` takes A LOT of calldata, most of it is ignored/useless when we are not actually executing an ERC-4337 useroperation. When doing a `multicall`, you want a simpler function to call.

Reply:
Closed: using ERC-7821 instead